Description: Set default DeviceTimeout to 5.0, otherwise boot fails,
 the patch is needed because Ubuntu doesn't install the defaults.plymouth
 configuration but use alternatives instead
Author: Dimitri John Ledkov <xnox@ubuntu.com>
Upstream: https://gitlab.freedesktop.org/plymouth/plymouth/merge_requests/44
---
 src/main.c |    4 ++++
 1 file changed, 4 insertions(+)

Index: plymouth-0.9.4git20190712/src/main.c
===================================================================
--- plymouth-0.9.4git20190712.orig/src/main.c
+++ plymouth-0.9.4git20190712/src/main.c
@@ -2273,6 +2273,10 @@ main (int    argc,
         find_system_default_splash (&state);
         find_distribution_default_splash (&state);
 
+        /* Device timeout may not be NAN or zero */
+        if (isnan (state.device_timeout) || state.device_timeout <= 0.0)
+                state.device_timeout = 5.0;
+
         if (ply_kernel_command_line_has_argument ("plymouth.ignore-serial-consoles"))
                 device_manager_flags |= PLY_DEVICE_MANAGER_FLAGS_IGNORE_SERIAL_CONSOLES;
 
